tests/integration/targets/azure_rm_aks/tasks/minimal-cluster.yml

```
@@ -0,0 +1,73 @@
+- set_fact:
+    rpfx: "{{ resource_group | hash('md5') | truncate(8, True, '') }}"
+    noderpfx: "{{ resource_group | hash('md5') | truncate(4, True, '') }}"
+
+- name: Use minimal parameters and system-assigned identity
+  azure_rm_aks:
+    name: "minimal{{ rpfx }}"
+    location: eastus
+    resource_group: "{{ resource_group }}"
```

Please add "kubernetes_version" parameter when create kubernets service! Thank you very much!
```suggestion
    resource_group: "{{ resource_group }}"
    kubernetes_version: **
```